oml/lightning/modules/retrieval.py

```diff
@@ -43,6 +44,7 @@ def __init__(
             labels_key: Key to get labels from the batches
             embeddings_key: Key to get embeddings from the batches
             scheduler_monitor_metric: Metric to monitor for the schedulers that depend on the metric value
+            freeze_n_epochs: number of epochs to freeze model (model should be instance of IFreezable)
```

Comment:
let's rephrase 
`(model should be instance of IFreezable)` -> `(for n > 0 model has to be an successor of IFreezable interface)`